apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/PictureChoiceSummary.tsx

```
@@ -44,43 +50,47 @@ export const PictureChoiceSummary = ({ questionSummary, survey, setFilter }: Pic
+            <button
+              className="w-full cursor-pointer hover:opacity-80"
+              key={result.id}
+              onClick={() =>
+                setFilter(
+                  questionSummary.question.id,
+                  questionSummary.question.headline,
+                  questionSummary.question.type,
+                  t("environments.surveys.summary.includes_all"),
+                  [`${t("environments.surveys.edit.picture_idx", { idx: index + 1 })}`]
+                )
+              }>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)